### PR TITLE
spell remote correctly

### PR DESCRIPTION
**Pull request by robrich**
- [X] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

File: javascript/node/selenium-webdriver/example/google_search.js [hunk 1]
```diff
@@ -47,4 +47,4 @@ driver.get('http://www.google.com/ncr');
 driver.findElement(By.name('q')).sendKeys('webdriver');
 driver.findElement(By.name('btnG')).click();
 driver.wait(until.titleIs('webdriver - Google Search'), 1000);
-driver.quit();
```
Comment (ddavison) [inline javascript/node/selenium-webdriver/example/google_search.js]: why are we removing the last line at EOF?

Comment (robrich) [inline javascript/node/selenium-webdriver/example/google_search.js]: Actually adding a CR after the last character, and not sure why GitHub's
web editor chose to do that. Feel free to merge that out or I can squash if
you'd like.

**Comment by robrich** [comment]
Removed the trailing CR, and squashed.

**Comment by ddavison** [comment]
merged in as f22d7c5.  thanks! :) :+1:

**Pull request closed without merging**